### ISSUE TITLE
iPad: README.md preview doesn’t scroll 

### ISSUE THREAD

**Issue by rogfrich** [gitpod-io/gitpod#1703]
### Describe the bug
I’m working on an iPad. When I open a particular workspace which has a README.md file, Theia opens a tab displaying a preview of the rendered Markdown file. This preview is too long to fit on the screen. Scrolling by dragging a finger up the screen on the tab does not scroll as expected. Instead, it appears to move a highlight to the left of each of the visible elements on the screen - a line, like the pipe symbol. You can see it in [this screenshot](https://imgur.com/wPKH8XE). This line jumps between Markdown elements when I drag, either with one finger or two. I am unable to scroll to see elements which are off the page.

### Steps to reproduce
- Use GitPod on an iPad (see below for detailed specs). Browser is iPadOS Safari.
- Open a repo in GitPod which has a README.md longer than can be displayed on an iPad screen
- Attempt to scroll through the preview which opens with the new workspace.

### Expected behaviour
I’d expect that the scroll gesture would scroll smoothly through the whole Markdown preview, like it would any other file.

### Additional information
Workspace ID: https://e554f9fa-a47e-4cae-94a3-9c77fb9054d9.ws-eu01.gitpod.io/#/workspace/soduku2
IPad specs: iPad Pro 11 2018, iPadOS 13.6. 
Issue happens in both landscape and portrait orientation.
Other files aren’t affected - other files, including the source README.md file, scroll smoothly as expected.

### Example repository
Workspace ID: https://e554f9fa-a47e-4cae-94a3-9c77fb9054d9.ws-eu01.gitpod.io/#/workspace/soduku2

**Comment by rogfrich** [comment]
Sorry, it was late last night when I posted this, and I missed out a potentially relevant bit of information: I've "installed" GitPod as a web app on my iPad's homepage and was running the site as a progressive web app, not a normal Safari page.

**Comment by jankeromnes** [comment]
Hi @rogfrich, thank you for reporting this issue. I was able to reproduce it on my iPad (most panels can be scrolled without issues, except the Markdown preview, or the Keyboard Shortcuts panel, which was already reported here: https://github.com/eclipse-theia/theia/issues/7763).

I'll take a look at both issues and see if they can be fixed in Theia. Then, once a fix is merged into Theia, we could roll it out to Gitpod within a few weeks.

**Comment by rogfrich** [comment]
Thanks @jankeromnes - Gitpod is the best solution I've found to doing proper development on the iPad, so it's great to hear this can be fixed. Thanks for all your efforts.

**Comment by C-alcium** [comment]
is there any plan to fix this?

**Comment by jankeromnes** [comment]
Hi @C-alcium, yes I'm still planning to work on this. I've already looked into it, but unfortunately my first suspicion wasn't verified, and I haven't been able to successfully debug this yet.

If you have any insight in debugging no-scroll issues on iPad, please let me know. Otherwise I'll try again soon.

**Comment by jankeromnes** [comment]
Also I think the Coder issue is about arrow keys and track pad, which have been fixed in webkit and already work well in Gitpod/Theia.

This issue is about any scrolling at all, in Markdown previews and Keybinding settings.

**Comment by C-alcium** [comment]
If I am able to carve out some time I'll look at it, but no promises at this time of year

**Comment by Dids** [comment]
Scrolling should be fixed in an upcoming iOS/iPadOS version, but no ETA on when yet. The WebKit part of the fix is [already in](https://trac.webkit.org/changeset/270712/webkit) though.

**Comment by C-alcium** [comment]
I see this is fixed in code-server -- is it fixed in gitpod?

**Comment by Dids** [comment]
> I see this is fixed in code-server -- is it fixed in gitpod? If so, this issue can be closed
> 
> Edit: I just tested it and it is not fixed.

"Fixed" is a strong word. From my own experience (and of a handful of others), it only works if you install a website as a PWA through Safari. Outside of that it doesn't seem to work.

**Comment by C-alcium** [comment]
> > I see this is fixed in code-server -- is it fixed in gitpod? If so, this issue can be closed
> > Edit: I just tested it and it is not fixed.
> 
> "Fixed" is a strong word. From my own experience (and of a handful of others), it only works if you install a website as a PWA through Safari. Outside of that it doesn't seem to work.

That's really unfortunate

**Comment by jankeromnes** [comment]
I don't think this issue is stale. We should keep it open until the problem is confirmed as fixed.

**Comment by jankeromnes** [comment]
@akosyakov @ghuntley do you know if Gitpod is still affected by this bug?

**Comment by akosyakov** [comment]
@jankeromnes 🤷‍♂️

**Comment by ghuntley** [comment]
Unable to reproduce on iPadOSb7. Closing it out. Please raise another issue if it manifests again.

https://user-images.githubusercontent.com/127353/131435617-4fed3cc9-733f-44f0-bf21-aad199316873.mp4
